pkg/config/template.go

```diff
@@ -90,8 +90,9 @@ type configuration struct {
 			Endpoint string
 			Token    string
 		}
-		LDAP *LDAP
-		OIDC OIDC
+		LDAP              *LDAP
+		OIDC              OIDC
+		LogoutRedirectURL string `mapstructure:"logout_redirect_url" validate:"logout_redirect_url"`
```

Comment:
```suggestion
		LogoutRedirectURL string `mapstructure:"logout_redirect_url"`
```

The _only_ supported validation is `validate:"required"`.  I think that this will be silently ignored.  (Reflection is a really nice way of hiding stuff...)

Reply:
oops